viewer/add-heading-anchors.js

```diff
@@ -143,6 +146,9 @@ var addHeadingAnchors = {
         window.location.hash = hash;
       }
     });
+  },
 
+  registerPostCopy: function(callback) {
+    $(this.target).find("a.headerLink").click(callback);
```

Comment:
Should we remove it? Alternatively, if you're going to keep it in, have you considered implementing this post-copy handling using events on the target object? This would eliminate the need for an explicit parameter to this init function:

The function here might look something like this:

```javascript
var $target = $(this.target);
$('a.headerLink', $target).click(function(e) {
   $target.trigger('post-link-copy.autoid', e);
});
```